release-notes/v1_10.md

```diff
@@ -256,6 +256,10 @@ Please note the following things:
 - This support is preliminary and subject to change.
 - The terminal task runner can now also be enabled by using the `"version": "2.0.0"` property.
 
+### New Predefined Variable to Facilitate Automation
```

Comment:
I recommend to change the title to be more descriptive of the actual work. It should definetely contain ${lineNumber}